pkg/actuators/machine/actuator.go

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright oVirt Authors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package machine
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-api-provider-ovirt/pkg/utils"
+	"time"
+
+	"k8s.io/client-go/rest"
+
+	machinev1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	apierrors "github.com/openshift/machine-api-operator/pkg/controller/machine"
+	"github.com/openshift/machine-api-operator/pkg/generated/clientset/versioned/typed/machine/v1beta1"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog"
+
+	osclientset "github.com/openshift/client-go/config/clientset/versioned"
+	ovirtconfigv1 "github.com/openshift/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
+	ovirtC "github.com/openshift/cluster-api-provider-ovirt/pkg/clients/ovirt"
+	ovirtsdk "github.com/ovirt/go-ovirt"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	TimeoutInstanceCreate       = 5 * time.Minute
+	RetryIntervalInstanceStatus = 10 * time.Second
+)
+
+// ActuatorParams holds parameter information for Actuator
```

Comment:
This comment is not very useful. Please consider removing or replacing it.

Reply:
Well, this comment is just to document the exported struct, I think it explains the purpose of it and it is the same on all providers.
Any suggestions?

Reply:
`ActuatorParams` is the data structure that contains the parameters required to set up a new `Actuator`. The following fields are required to initialize the `Actuator`: ...